collector/core/core.go

```diff
@@ -0,0 +1,24 @@
+// core.go
+
+package core
```

Comment:
I don't think it is necessary to introduce a new package considering there is only one file that functions to print versions. Even if we introduce it, I think the package name `core` is kind of vague since this package only contains the functions for printing versions.
Did you refer to any other open-source repositories for introducing a `core` package? I prefer to move this file just under the `collector` directory.